cmake/custom_cxxstandard.cmake

```diff
@@ -31,7 +31,7 @@ elseif (CMAKE_CXX_COMPILER_ID MATCHES Intel)
     file(REMOVE ${_testfl})
 
     if (APPLE)
-        if (${GCC_VERSION} VERSION_LESS 6.1)
+        if (${GCC_VERSION} VERSION_LESS 3.6)
             message(FATAL_ERROR "${BoldYellow}Intel ICPC makes use of CLANG (detected: ${GCC_VERSION}; required for C++11: 6.1) so this build won't work without CLANG intervention: http://psicode.org/psi4manual/master/build_planning.html\n${ColourReset}")
```

Comment:
You moved this back to 3.6, but didn't update the message. Can we really support GCC 3.6 now?

Reply:
Ah, whoops, I'll fix it to "C++11: 3.6". Don't be mislead by the variable names; in this case, `${GCC_VERSION}` contains Clang version. Maybe I should rename to CPLR_VERSION.